frontend/src/components/composed/ContactModal/ContactModal.js

```diff
@@ -62,20 +62,11 @@ const ContactModal = ({id, active, hideModal}) => {
 
 		if (isAnyFieldEmpty()) {
 			setError("Missing required fields");
-			return;
 		} else {
 			setError("");
+			setFormIsValid(true);
+			sendContact(nameValue, emailValue, messageValue);
 		}
-
-		sendContact(name, email, message, (error) => {
-			if (error) {
-				setError(error);
-			} else {
-				setError("");
-				console.log("The message has been sent!");
-				resetForm();
-			}
-		});
 	};
 
 	return (
```

Comment:
Include the inputs inside a form and hook to the submit event of the form to validate the whole schema with YUP